_posts/2020-08-10-making-eslint-more-inclusive.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Making ESLint more inclusive"
+tags:
+  - Inclusivity
+  - Diversity
+  - Code of Conduct
+---
+
+# Making ESLint more inclusive
+
+From its beginning, the ESLint project has sought to be a place that is welcoming and inclusive of everyone. We adopted a code of conduct before doing so was common (even before we joined the jQuery Foundation, now the Open JS Foundation) and the project has been maintained by a group of developers from all around the world, representing a variety of ethnicities, cultures, and countries. Even so, we understand that inclusivity is a process that must be adapted as we learn and grow. There is always some way to be more welcoming, and today we're happy to share with you some of our recent efforts.
+
+## Eliminating problematic terminology
+
+One of the areas where we've made changes in the project is through the use of more inclusive terminology. We recently did an audit of our codebase to look for words with negative connotations. In doing so, we discovered that we had a lot of instances of the word "blacklist" in the project, not just inside of code but also in the name of the `id-blacklist` rule.
```

Comment:
I think "a few" more accurately represents how many we had to remove (see https://github.com/eslint/eslint/commit/004adae3f959414f56e44e5884f6221e9dcda142).

```suggestion
One of the areas where we've made changes in the project is through the use of more inclusive terminology. We recently did an audit of our codebase to look for words with negative connotations. In doing so, we discovered that we had a few instances of the word "blacklist" in the project, not just inside of code but also in the name of the `id-blacklist` rule.
```